### PR TITLE
Tweaks to get sockets working on FE

### PR DESCRIPTION
**Pull request by damian-pf**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -38,7 +17,5 @@ redis.on('message', function(channel, rawMessage) {
     io.emit(channel, message.data);
 });
 
-
-app.listen(PORT, function() {
-    console.log('Example app listening on port 3000!');
-});
+console.log('Good to go bro!');
```
Comment (titopixelfusion) [inline app.js]: Go where?! :)
I'd remove a comment/log that doesn't answer the question 'why'

File: app.js [hunk 2]
```diff
@@ -1,31 +1,10 @@
-const express = require('express');
-const timeout = require('connect-timeout');
 const server = require('http').Server();
-const app = express();
-
-const PORT = 3000;
-
-const io = require('socket.io')(server);
+const io     = require('socket.io').listen(server);
+const PORT   = 3001;
```
Comment (titopixelfusion) [inline app.js]: why change the port number?

File: app/Http/Controllers/Controller.php [hunk 3]
```diff
@@ -63,6 +63,7 @@ public function joinRoom(Request $request)
         $persons = Person::where(['room_id' => $roomId])->with('orders')->get();
 
         foreach ($persons as &$person) {
+            $person->person_id = $person->id;
```
Comment (titopixelfusion) [inline app/Http/Controllers/Controller.php]: why not just $person->id? Seem redundant to have $person->person_id

File: app.js (hunk 2 above)
Comment (damian-pf) [reply app.js]: For local dev. The JS app runs at 3000

File: app/Http/Controllers/Controller.php (hunk 3 above)
Comment (damian-pf) [reply app/Http/Controllers/Controller.php]: For the front end. As we dont have a model structure we have the ids all prefixed

**Pull request merged**